test/nodes/test_search_hybrid.py

```
@@ -0,0 +1,749 @@
+        depends_mod = types.ModuleType('depends')
+        depends_mod.depends = lambda *a, **k: None
+
+        sys.modules['ai'] = ai_pkg
+        sys.modules['ai.common'] = ai_common
+        sys.modules['ai.common.config'] = config_mod
+        sys.modules['ai.common.schema'] = schema_mod
+        sys.modules['depends'] = depends_mod
+
+        return saved, stub_names, schema_mod, SubQuestion
+
+    def _restore_modules(self, saved, stub_names):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_restore_modules`

Add return type annotation: `None`

(ANN202)